osvr_tracked_device.h

```diff
@@ -488,7 +422,7 @@ const char* OSVRTrackedDevice::GetSerialNumber()
 
 float OSVRTrackedDevice::GetIPD()
 {
-    return static_cast<float>(m_DisplayConfiguration->getIPDMeters());
+    return 0.063f; /// @todo update when this becomes exposed to the API
```

Comment:
@rpavlik came up with the idea to use the distance between the two eye poses for the IPD, instead of hard coding it. That way it is a longer term solution.